12-add-tupletools.md

```diff
@@ -87,22 +87,10 @@ The usage of `Branches` is very important (and strongly encouraged) to keep the
 > ## Where to find TupleTools {.callout}
 > One of the most difficult things is to know which tool we need to add to our `DecayTreeTuple` in order to get the information we want.
 > For this, it is necessary to know where to find `TupleTools` and their code.
-> `TupleTools` are spread in 9 packages under `Analysis/Phys` (see the head of `svn` [here](https://svnweb.cern.ch/trac/lhcb/browser/Analysis/trunk/Phys)), all starting with the prefix `DecayTreeTuple`, according to the type of information they fill in our ntuple:
 >
-> - `DecayTreeTuple` for the more general tools.
-> - `DecayTreeTupleANNPID` for the NeuralNet-based PID tools.
-> - `DecayTreeTupleDalitz` for Dalitz analysis.
-> - `DecayTreeTupleJets` for obtaining information on jets.
-> - `DecayTreeTupleMC` gives us access to MC-level information.
-> - `DecayTreeTupleMuonCalib` for muon calibration tools.
-> - `DecayTreeTupleReco` for reconstruction-level information, such as `TupleToolTrackInfo`.
-> - `DecayTreeTupleTracking` for more detailed tools regarding tracking.
-> - `DecayTreeTupleTrigger` for accessing to the trigger information of the candidates.
+> `TupleTools` are Gaudi tools that implement the [`IParticleTupleTool`](http://lhcb-release-area.web.cern.ch/LHCb-release-area/DOC/davinci/releases/latest/doxygen/d1/d77/class_i_particle_tuple_tool.html) interface (if they fill information about an `LHCb::Particle`) or the [`IEventTupleTool`](http://lhcb-release-area.web.cern.ch/LHCb-release-area/DOC/davinci/releases/latest/doxygen/d7/d5c/class_i_event_tuple_tool.html) interface (if they fill information about the event in general), so we can get the complete list of `TupleTools` of each type at the top of the doxygen page of the interface.
```

Comment:
Instead of getting rid of the existing lines I would add this after. For `TupleTools` we can link to doxygen but getting people used to looking directly in svn is IMO very valuable.

We might need to add a callout box explaining in very brief terms what interfaces, Gaudi tools, etc are as none of these concepts have been explained yet.